jest.config.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
+  preset: 'ts-jest/presets/js-with-babel',
   testEnvironment: 'jsdom',
   verbose: true,
-  snapshotSerializers: ['@emotion/snapshot-serializer'],
+  snapshotSerializers: ['@emotion/jest/enzyme-serializer'],
```

Comment:
I noticed emotion recommends this serializer when using enzyme, so I switched it while I was at it.